content/learning-paths/servers-and-cloud-computing/onnx-on-azure/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Deploy SqueezeNet 1.0 INT8 (opset 12) model with ONNX Runtime on Azure Cobalt 100
+
+minutes_to_complete: 60   
+
+who_is_this_for: This is an introductory topic for the software developers who are willing to migrate their ONNX based applications from x86_64 platforms to ARM based platforms, or on Microsoft Azure - Cobalt 100 CPU based VMs specifically.  
+
+learning_objectives: 
```

Comment:
We discussed detailed points about azure linux docker and vm image in Java PR. Follow that here, as weel as in all azure specific PRs.